include/aspect/compat.h

```diff
@@ -28,4 +28,70 @@
 #include <functional>
 #include <memory>
 
+
+
+namespace big_mpi
+{
+
+#if DEAL_II_VERSION_GTE(10,0,0)
+
+  template <typename T>
+  void
+  broadcast(T                 *buffer,
+            const size_t       count,
+            const unsigned int root,
+            const MPI_Comm    &comm)
+  {
+    dealii::Utilities::MPI::broadcast(buffer, count, root, comm);
+  }
```

Comment:
I think it would be easier to know what we need to do once we rely on deal.II 10.0 if you just did
```suggestion
  using dealii::Utilities::MPI::broadcast;
```